cinema/serializers.py

```diff
@@ -1 +1,102 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (CinemaHall,
+                           Genre,
+                           Actor,
+                           Movie,
+                           MovieSession
+                           )
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    capacity = serializers.ReadOnlyField()
```

Comment:
The `capacity` field is set as `ReadOnlyField`. If `capacity` is already a field in the `CinemaHall` model, this might be redundant. Consider removing it unless there's a specific reason to override it as read-only.